test-packages/01-basic-app/app/models/foo.js

```diff
@@ -0,0 +1 @@
+export default class Foo {}
```

Comment:
per https://github.com/miragejs/ember-cli-mirage/commit/3cd81ef7ee3f67e5edfaf725cf64747ba2313183#r129177371 agree with @cah-brian-gantzler that this is not supposed to work as it's not Ember Data model